src/installation/liveimages/guide-chroot.md

```diff
@@ -0,0 +1,388 @@
+# mount -t sysfs sys /mnt/sys
+# mount -o bind /dev /mnt/dev
+# mount -t devpts pts /mnt/dev/pts
+```
+
+Copy the `/etc/resolv.conf` file to the new system, so that DNS addresses can be
+resolved.
+
+```
+# cp -L /etc/resolv.conf /mnt/etc/
+```
+
+Finally, change root into the new system, and source the new shell profile. You
+can also change the prompt temporarily to remind yourself  while you're inside
+the chroot jail.
+
+```
+# chroot /mnt /bin/bash
+# . /etc/profile
+# export PS1="(chroot) \w \\$ "
+```
+
+## Configure the system
+
+### Root user
+
+Create a password for the root user, and set the correct permissions.
+
+```
+# passwd root
+# chown root:root /
+# chmod 755 /
+```
+
+### Fstab
+
+Add a line in `/etc/fstab` for each partition that should be mounted at boot
+time, as well as the `swap` partition to be enabled. For mount options,
+`defaults` and `noatime` are safe to enable. If you use a Solid State Drive,
+you can also enable the `discard` option for continuous TRIM and will help with
+sustained long-term performance and wear-leveling. See
+[`fstab(5)`](https://man.voidlinux.org/fstab.5) for more information on this
+file and what options are available.
+
+```
+#
+# See fstab(5).
+#
+# <file system> <dir>      <type> <options>                 <dump>  <pass>
+tmpfs           /tmp       tmpfs  defaults,nosuid,nodev     0       0
+/dev/sda1       swap       swap   defaults,noatime,discard  0       0
+/dev/sda2       /boot/efi  vfat   defaults,noatime,discard  0       2
+/dev/sda3       /          ext4   defaults,noatime,discard  0       1
+/dev/sda4       /home      ext4   defaults,noatime,discard  0       2
+```
+
+### Hostname
+
+Set a hostname to identify your machine on the network. See
+[`hostname(1)`](https://man.voidlinux.org/hostname.1) and
+[`hostname(7)`](https://man.voidlinux.org/man7/hostname.7) for more information.
+
+```
+# echo <hostname> > /etc/hostname
+```
+
+### rc.conf
+
+Edit the `/etc/rc.conf` file. Here you should uncomment and set the
+`HARDWARECLOCK`, `TIMEZONE`, and `KEYMAP` variables. Some other options can be
+set here as well.
+
+```
+# Set RTC to UTC or localtime.
+HARDWARECLOCK="UTC"
+
+# Set timezone, availables timezones at /usr/share/zoneinfo.
+TIMEZONE="America/Los_Angeles"
+
+# Keymap to load, see loadkeys(8).
+KEYMAP="dvorak"
+```
+
+### Localization
+
+If using the *glibc* variant, you must set the locale. This step should be
+skipped if using the *musl-libc* variant.
+
+Edit the `/etc/default/libc-locales` file and uncomment the `en_US.UTF-8` locale
+and any other locales that should be made available.
+
+```
+...
+en_US.UTF-8 UTF-8
+en_CA.UTF-8 UTF-8
+...
+```
+
+Then, force a reconfiguration of the `glibc-locales` package:
+
+```
+# xbps-reconfigure --force glibc-locales
+```
+
+### Boot loader
+
+Install grub onto the EFI partition:
+
+```
+# grub-install --target=x86_64-efi --efi-directory=/boot/efi --bootloader-id=void-grub --boot-directory=/boot --recheck
+```
+
+## Reboot
```

Comment:
I could have sworn you have to generate the initramfs before rebooting, but maybe I'm wrong.

Reply:
Forgot that, I don't think it does generate one automatically. Will have to test that :thinking:  anyways, added that in.

Reply:
I just tested and it looks like it does in fact generate an initramfs. So I'll remove the section but it would be a good thing to add in another part of the docs